spec/restforce_mock_spec.rb

```diff
@@ -68,14 +104,29 @@
       end
 
       it "validates required fields" do
-        RestforceMock::Sandbox.add_required(
-          "Object__c", [:Section_Name__c, :Program__c ])
+        RestforceMock.configure do |config|
+          config.schema_file = "spec/fixtures/required_schema.yml"
+        end
 
         values = { Name: "Name here" }
         expect {
           client.api_post("/sobjects/Object__c", values)
-        }.to raise_error Faraday::ResourceNotFound, /Required fields are missing/
+        }.to raise_error Faraday::ResourceNotFound,
+        /REQUIRED_FIELD_MISSING: Required fields are missing: \["Id\", \"Program__c\", \"Section_Name__c\"\]/
```

Comment:
Line is too long. [109/80]<br>Align the parameters of a method call if they span more than one line.